models/event_argument_extraction/KnowledgeExtractor.py

```diff
@@ -0,0 +1,79 @@
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+
+class KnowledgeExtractor(object):
+    def __init__(self):
+        self.path_to_ckpt = ''
+        self.tokenizer = None
+        self.model = None
+        self.model_para = dict()
+
+    def instantiate(self):
+        pass
+         
+    def forward(self):
+        pass
```

Comment:
```suggestion
class KnowledgeExtractor(object):
    def __init__(self, path_to_ckpt: str):
        self.tokenizer, self.model = self.instantiate(path_to_ckpt)
        
    @abstractmethod
    def instantiate(path_to_ckpt: str):
        pass
         
    def forward(self, text: Union[Text, List[Text]]):
        pass
```

Reply:
union method? it would raise an error right?

Reply:
will do! :p